common/api-review/auth.api.md

```diff
@@ -647,6 +653,9 @@ export interface ReactNativeAsyncStorage {
     setItem(key: string, value: string): Promise<void>;
 }
 
+// @public
+export const reactNativeLocalPersistence: Persistence;
```

Comment:
Does `getReactNativePersistence` also need to be exported here? I think it's exported out of index.doc.d.ts alongside `reactNativeLocalPersistence`

Reply:
Just removed this file from the PR.